clusterloader2/testing/load/config.yaml

```diff
@@ -1,36 +1,40 @@
+# ASSUMPTIONS:
+# - Underlying cluster should have 100+ nodes.
+# - Number of nodes should be divisible by NODES_PER_NAMESPACE (default 100).
+
 #Constants
 {{$BIG_GROUP_SIZE := 250}}
 {{$MEDIUM_GROUP_SIZE := 30}}
 {{$SMALL_GROUP_SIZE := 5}}
-{{$PODS_PER_NODE := 30}}
 {{$NODES_PER_NAMESPACE := 100}}
-#Variables
+{{$PODS_PER_NAMESPACE := 3000}}
```

Comment:
The constant to use should be "PODS_PER_NODE" (also in density test).

This is the thing that is better understood.

Reply:
Changed to `PODS_PER_NODE`.

Reply:
"PODS_PER_NODE" in density test - #280